client/views/windows/settings.tpl

```diff
@@ -5,6 +5,19 @@
 	<h1 class="title">Settings</h1>
 
 	<div class="row">
+		{{#unless public}}
+		<div class="col-sm-12">
+			<h2>
+				Settings synchronisation
+			</h2>
+			<p class="opt">
+				<input type="checkbox" name="syncSettings">
```

Comment:
Make this a correct `<label>` so the text is also clickable. (look at other checkboxes)

Don't include warnings as part of the label though.

Reply:
I explicitly didn't do that as that creates a bunch of issues with settings styling and how things are aligned and you really don't want the text to be part of the label as you don't want to accidentally activate this. 

Also just having the input element in a label doesn't make much sense to me?

Reply:
Done